src/main/java/com/twilio/http/HttpUtility.java

```diff
@@ -0,0 +1,32 @@
+package com.twilio.http;
+
+import com.twilio.Twilio;
+import lombok.experimental.UtilityClass;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@UtilityClass
+class HttpUtility {
+    String getUserAgentString(final List<String>... userAgentExtensions) {
```

Comment:
I don't think we should accept a vararg as input. Simpler to just accept a single `List`.

Reply:
addressed. I have removed varargs completely from the code.